wagtail/contrib/search_promotions/templates/wagtailsearchpromotions/queries/chooser_field.js

```diff
@@ -3,7 +3,7 @@ function createQueryChooser(id) {
   var input = $('#' + id);
 
   chooserElement.on('click', function () {
-    var initialUrl = '{% url "wagtailsearch_admin:queries_chooser" %}';
```

Comment:
question: do we need to document the fact that the chooser has changed?
It seems unlikely anyone is doing anything fancy with search promotions, but in case they've customised something...

Reply:
I take it back. you explicitly call this out in the ticket description!

Reply:
I'll include a cleaned up version of those upgrade considerations when I merge. Thanks @zerolab